midtransclient/tokenization.py

```diff
@@ -0,0 +1,87 @@
+from .config import ApiConfig
+from .http_client import HttpClient
+
+class Tokenization:
+    """
+    Link the customer's account to be used for payments using specific payment channel
+    (more params detail refer to: https://api-docs.midtrans.com/#gopay-tokenization)
+    """
+
+    def __init__(self, 
+            is_production=False,
+            server_key='',
+            client_key='',
+            custom_headers=dict(),
+            proxies=dict()):
+
+        self.api_config = ApiConfig(is_production,server_key,client_key,custom_headers,proxies)
+        self.http_client = HttpClient()
+
+    @property
+    def api_config(self):
+        return self.__api_config
+
+    @api_config.setter
+    def api_config(self, new_value):
+        self.__api_config = new_value
+
+    def link_account(self,parameters=dict()):
+        """
+        Trigger `/v2/pay/account` API call to Core API
```

Comment:
method names should be consistent with other library:
- https://github.com/veritrans/veritrans-ruby/blob/master/lib/veritrans/api.rb#L174-L251
- https://github.com/Midtrans/midtrans-go/blob/main/coreapi/clienttokenization.go